client/app/components/groups/ListItemAddon.jsx

```diff
@@ -4,16 +4,34 @@ import Tooltip from "antd/lib/tooltip";
 
 export default function ListItemAddon({ isSelected, isStaged, alreadyInGroup, deselectedIcon }) {
   if (isStaged) {
-    return <i className="fa fa-remove" />;
+    return (
+      <>
+        <i className="fa fa-remove" aria-hidden="true" />
+        <span className="sr-only">Remove</span>
+      </>
+    );
   }
   if (alreadyInGroup) {
     return (
       <Tooltip title="Already selected">
-        <i className="fa fa-check" />
+        <span tabIndex={0}>
```

Comment:
What's the tab index for? Is that for triggering the tooltip?

Reply:
Yup. Otherwise it's impossible to trigger it via focus. A further PR will globally add focus as a trigger for tooltips.